tools/clang_tidy/lib/clang_tidy.dart

```diff
@@ -89,8 +89,12 @@ class ClangTidy {
   final StringSink _outSink;
   final StringSink _errSink;
 
+  DateTime? _startTime;
```

Comment:
Could probably be `late DateTime _startTime` since that's how it's used.

Reply:
done